src/ci/scripts/install-clang.sh

```diff
@@ -12,10 +12,19 @@ source "$(cd "$(dirname "$0")" && pwd)/../shared.sh"
 LLVM_VERSION="10.0.0"
 
 if isMacOS; then
-    curl -f "${MIRRORS_BASE}/clang%2Bllvm-${LLVM_VERSION}-x86_64-apple-darwin.tar.xz" | tar xJf -
+    # If the job selects a specific Xcode version, use that instead of
+    # downloading our own version.
+    if [[ -s "${SELECT_XCODE}" ]]; then
+        sudo xcode-select -s "${SELECT_XCODE}"
+        bindir="${SELECT_XCODE}/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin"
```

Comment:
Following up on the "split this into two environment variables", we can avoid relying on `SELECT_XCODE` to find clang:

```suggestion
        bindir="$(xcode-select --print-path)/Toolchains/XcodeDefault.xctoolchain/usr/bin"
```